src/platform/packages/shared/kbn-esql-validation-autocomplete/src/autocomplete/helper.ts

```diff
@@ -112,14 +112,17 @@ export function removeQuoteForSuggestedSources(suggestions: SuggestionRawDefinit
   }));
 }
 
+const leftHandSideParamName = new Set(['left', 'field', 'lhs']);
```

Comment:
Haven't we mapped in the script above as left? Why do we need them here?

Reply:
Good point! This was a left over. I removed it here [`9f938ef` (#205565)](https://github.com/elastic/kibana/pull/205565/commits/9f938ef0c4364a1e386daefdb9b6a9cf25c27a23)